release/releaser.go

```diff
@@ -582,11 +589,19 @@ func (r *releaser) releaseActionRegradeServices(services []flux.ServiceID, msg s
 				}
 
 				namespace, serviceName := service.Components()
-				rc.Instance.LogEvent(namespace, serviceName, "Starting regrade "+cause)
-				specs = append(specs, platform.RegradeSpec{
-					ServiceID:     service,
-					NewDefinition: def,
-				})
+				if serviceName == FluxServiceName {
+					rc.Instance.LogEvent(namespace, serviceName, "Starting regrade (no result expected) "+cause)
+					asyncSpecs = append(asyncSpecs, platform.RegradeSpec{
+						ServiceID:     service,
+						NewDefinition: def,
+					})
+				} else {
+					rc.Instance.LogEvent(namespace, serviceName, "Starting regrade "+cause)
+					specs = append(specs, platform.RegradeSpec{
+						ServiceID:     service,
+						NewDefinition: def,
+					})
+				}
```

Comment:
Might be cleaner if `Async: true`, were part of the spec? But I guess we want to do all the sync spec, *then* any async ones after, so... fine.